2996_reflection/reflection.md

```diff
@@ -4969,7 +4938,7 @@ using alias = [:^^S::type:];    // OK, type-only context
 
 [#]{.pnum} For a `$splice-type-specifier$` of the form `typename@~_opt_~@ $splice-specifier$`, the `$splice-specifier$` shall designate a type, a primary class template, or an alias template. The `$splice-type-specifier$` designates the same entity as the `$splice-specifier$`.
 
-[#]{.pnum} For a `$splice-type-specifier$` of the form `typename@~_opt_~@ $splice-specialization-specifier$`, the `$splice-specifier$` of the `$splice-specialization-specifier$` shall designate a primary class template or an alias template. The `$splice-type-specifier$` designates the specialization of `$T$` corresponding to the `$template-argument-list$` (if any) of the `$splice-specialization-specifier$`.
+[#]{.pnum} For a `$splice-type-specifier$` of the form `typename@~_opt_~@ $splice-specialization-specifier$`, the `$splice-specifier$` of the `$splice-specialization-specifier$` shall designate a template `$T$` that is either a primary class template or an alias template. The `$splice-type-specifier$` designates the specialization of `$T$` corresponding to the template argument list of the `$splice-specialization-specifier$`.
```

Comment:
Why "template argument list" instead of `$template-argument-list$`, don't we want the grammar term?

Reply:
We made an analogous change in the last telecon [here](https://brevzin.github.io/cpp_proposals/2996_reflection/d2996r13.html#pnum_151). The consensus seemed to be that while the `$template-argument-list$` might be absent (e.g., `f<>`), there is still a template argument list (even if it's empty). 🤷